make/migrations/postgresql/0030_1.11.0_schema.up.sql

```diff
@@ -0,0 +1,37 @@
+/* TODO remove the table artifact_2 and use the artifact instead after finishing the upgrade work */
+CREATE TABLE artifact_2
+(
+  id            SERIAL PRIMARY KEY NOT NULL,
+  /* image, chart, etc */
+  type          varchar(255),
+  project_id    int NOT NULL,
+  repository_id int NOT NULL,
+  media_type    varchar(255),
+  digest        varchar(255) NOT NULL,
+  size          int,  
```

Comment:
the `size` is stand for the artifact itself or the sum of its blobs?

Reply:
The `Artifact` is the concept of Harbor, not the manifest or index, so it is the size of image or chart